src/sentry/static/sentry/app/views/releases/releaseOverview.jsx

```diff
@@ -202,6 +221,28 @@ const ReleaseOverview = React.createClass({
               })
             }
           </ul>
+          <h6 className="nav-header m-b-1">{t('Deploys')}</h6>
+          <ul className="nav nav-stacked">
+            { !deploys.length ? this.renderEmpty() :
+              deploys.map((deploy) => {
+                return (
+                  <li key={deploy.id}>
+                    <a href={`/${orgId}/${projectId}/?query=environment:${deploy.environment}+release:${version}`}>
```

Comment:
You need to [encode these](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) for use in a query string:

```javascript
${encodeURIComponent(deploy.environment)}
```

Also ... should `query` not also be quoted?

```javascript
<a href={`/${orgId}/${projectId}/query="environment:${encodeURIComponent(deploy.environment)}+release:${encodeURIComponent(version_}"`
```